changelog/fragments/1669236059-Capture-stdout-stderr-of-all-spawned-components-to-simplify-logging.yaml

```diff
@@ -0,0 +1,31 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
+
+# Change summary; a 80ish characters long description of the change.
+summary: Capture stdout/stderr of all spawned components to simplify logging
```

Comment:
I think we should mention that the default log level for Beat sub-processes is not longer debug.

Reply:
That is not going to fit in the 80ish character limit. Should I add that as another changelog entry? Or just make it really long?

Reply:
Just got it to fit in the same one.

Reply:
You can use the description field if you need it to be longer than 80 characters.